src/requirements.txt

```diff
@@ -1,5 +1,5 @@
-anki==2.1.32
-ankirspy==2.1.32
+anki==2.1.36
```

Comment:
Nice. But this requirements.txt is a generated file and the version change should actually be done in the pyproject.toml. I've opened a new issue related to this here: https://github.com/ankicommunity/anki-sync-server/issues/41.